packages/@azure/eventhubs/client/changelog.md

```diff
@@ -1,3 +1,16 @@
+### 2019-03-26 2.0.0
+
+#### Breaking Changes
+- If you have been using the `createFromAadTokenCredentials` function to create an instance of the `EventHubClient`, you will now need to use the [@azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) library instead of [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) library to create the credentials that are needed by the `createFromAadTokenCredentials` function.
+- If you have been passing a non string value in the `partitionKey` property on the message when sending it using the `EventHubClient`, an error will now be thrown. This property only supports string values.
+
+#### Bug fixes
+- A network connection lost error is now treated as retryable error. A new error with name `ConnectionLostError` is introduced for this scenario which you can see if you enable the logs.
+- When recovering from an error that caused the underlying AMQP connection to get disconnected, [rhea](https://github.com/amqp/rhea/issues/205) reconnects all the older AMQP links on the connection resulting in the below 2 errors in the logs. We now clear rhea's internal map to avoid such reconnections. We already have code in place to create new AMQP links to resume send/receive operations.
```

Comment:
It isn't clear from reading this what the change was on our side.

Reply:
> We now clear rhea's internal map to avoid such reconnections.

?

Reply:
I was confused by the next statement.

> We already have code in place to create new AMQP links to resume send/receive operations.

Reply:
I didn't want "We now clear rhea's internal map to avoid such reconnections." to mislead users into thinking we will never reconnect.

What I am trying to convey is that we (the library) has always had the code in place to create new AMQP links to resume operations. We stopped rhea from reconnecting old links that caused the errors in the logs